content/blog/release-0.11.0.md

```diff
@@ -0,0 +1,137 @@
+
+### Hardware based testing
+
+We have recently merged a great addition by [@Tiwalun](https://github.com/Tiwalun) to improve testing of our PRs in the form of a smoke testing tool in the main repository.
+You can set it up to test each PR against the devkits you own. This will catch many regressions and errors before we can merge them into master.
+
+To step this one up, we have worked on a [custom hardware rig](https://github.com/probe-rs/hive) which will allow us to do large scale automated testing of each PR on a set of N targets multiplexed against M debug probes.
+A sneak peek of the rigs design can be seen below ;)
+
+<center><img src="/img/release-0.11.0/hive.jpg" style="max-width:100%"></center>
+
+To push this project a little further, I have put up a posting for a BSc theory with a local technical university (Switzerland). If you are a BSc student and would like to work on this, feel free to reach out to [@Yatekii](https://github.com/Yatekii) no matter of locality. I cannot promise anything but I will try to figure something out to make this work!
+
+## Various
+
+### hs-probe
+
+We announced the [hs-probe](https://github.com/probe-rs/hs-probe) quite a while ago. Many of you have ordered one already. Unfortunately the manufacturing was hit by the global chip shortages as well. The good news is that on the 26th of June the manufacturing should conclude and we can start the shipping of the probes!
+[Emil](https://github.com/korken89) has been working really hard to make this a possibility and get the probe manufactured even during chip shortages!
+If you have not ordered a probe yet, and would like to get ahold one, you can do so on our [shop](https://shop.probe.rs/).
+
+### Automatic releases
+
+We now have a pipeline set up to automate releases of the probe-rs core library and related crates. This should allow us to release way more often so your own tools do not have to pull from master constantly.
+At the start we are aiming for approximately one release every two weeks. This is an experiment and we will see how it goes. We believe that we should bring new functionality and especially fixes to you as soon as possible.
+
+Alongside more releases I'd like to try to blog a little more about the ongoing development and curiosities we encounter. Though, to focus more on development and user support I will keep this promise as small as possible for now ;)
+
+### Sponsoring
+
+We put a lot of work into probe-rs, because we believe in it being an important cornerstone to the embedded development ecosystem.
+All of this is voluntary work and we continue to do so.
+But during times when the life at our dayjob is a little more stressful it can become hard to spend the evening putting in extra work hours, which is why we set up a sponsoring possibility.
```

Comment:
```suggestion
All of this is voluntary work and we continue to work on it voluntarily, but during times when life at our day-job is a little more stressful it can become hard to spend the evening putting in extra work hours, which is why we set up a sponsorship option.
```